### DISCUSSION TITLE
Hyperspace with background image

### DISCUSSION THREAD

**Discussion by returntowholeness** [matteobruni/tsparticles#4244] [General]
I'm trying to add a background image with the emit center effects from the hyperspace example.  Matteo had responded in another post that it's not possible.  Wonder if there a hack or work around to make it work? 

Thanks! Vincent

**Comment by matteobruni** [comment]
The hyperspace sample you can see in the website is using move trails, that fills the canvas with the provided fillColor property. The trail is made by a semitransparent layer, that's why the image disappears. If you disable trails, you can see the image correctly.

**Reply by returntowholeness** [reply]
thanks that worked!
